t/t7502-commit-porcelain.sh

```diff
@@ -262,13 +262,57 @@ test_expect_success 'cleanup commit messages (scissors option,-F,-e, scissors on
 
 	echo >>negative &&
 	cat >text <<-\EOF &&
+
 	# ------------------------ >8 ------------------------
 	to be removed
 	EOF
 	git commit --cleanup=scissors -e -F text -a --allow-empty-message &&
 	git cat-file -p HEAD >raw &&
 	sed -e "1,/^\$/d" raw >actual &&
-	test_must_be_empty actual
+	test_must_be_empty actual 
+'
+
+test_expect_success 'cleanup commit messages (scissors option,-F,-e, CR/LF line endings)' '
+
+	echo >>negative &&
+	test_config core.editor unix2dos &&
```

Comment:
That assumes that `unix2dos` is installed. Which is already not in the case _at least_ in our `GETTEXT_POISON` job of the PR build:

```
2020-07-01T00:49:32.0212633Z error: cannot run unix2dos: No such file or directory
```

I wonder why you preferred `unix2dos` over the method I proposed (which I based on the shell function [`append_cr()`](https://github.com/git-for-windows/git/blob/v2.27.0.windows.1/t/test-lib-functions.sh#L111-L113) that Git's test suite _already_ has, and that is therefore well-tested and robust)?

Reply:
I've changed the test to `test_config core.editor "\"$PWD/lf-to-crlf.sh\""` to get local tests working in git-bash.